Include only <glib.h>. Fixes:

error: #error "Only <glib.h> can be included directly."

https://sourceforge.net/tracker/?group_id=213712&atid=1026806&aid=3600646
https://sourceforge.net/p/gelide/bugs/19/

Change 1 of 6 from
https://sourceforge.net/p/gelide/code/ci/b8da37d825b9548ba6548f96ce8efb84cf068001/
--- src/core/system.cpp
+++ src/core/system.cpp
@@ -21,7 +21,7 @@
 
 #include <fstream>
 #include <algorithm>
-#include <glib/gstdio.h>
+#include <glib.h>
 #include <sstream>
 #include "system.hpp"
 #include "../utils/tokenizer.hpp"
--- src/core/system_manager.cpp
+++ src/core/system_manager.cpp
@@ -22,7 +22,7 @@
 #include <fstream>
 #include <stdlib.h>
 #include <sstream>
-#include <glib/gstdio.h>
+#include <glib.h>
 #include <gtkmm/image.h>
 #include <giomm/file.h>
 #include "system_manager.hpp"
--- src/utils/utils.cpp
+++ src/utils/utils.cpp
@@ -21,7 +21,7 @@
 #include "utils.hpp"
 #include "../gelide.hpp"
 #include <ctime>
-#include <glib/gfileutils.h>
+#include <glib.h>
 //#include <glibmm.h>
 //#include <glib/gstdio.h>
 //#include <stdlib.h>
